### DISCUSSION TITLE
how to compile .c for termux with github actions ?

### DISCUSSION THREAD

**Discussion by offici5l** [termux/termux-packages#20955] [Q&A] [answered]
how to compile .c for termux with github actions ?

**Comment by sylirre** [comment, marked as answer]
Use Android NDK. This is what being used to compile the packages.

**Comment by offici5l** [comment]
tks for the help 🙏 

https://github.com/offici5l/MiAssistantTool/blob/main/.github%2Fworkflows%2Fbuild.yml#L51-L64
